tidy3d/web/api/tidy3d_stub.py

```diff
@@ -0,0 +1,164 @@
+"""Stub for webapi"""
+from __future__ import annotations
+
+import json
+from typing import Union
+
+from pydantic.v1 import BaseModel
+
+from ..core.file_util import read_simulation_from_hdf5, read_simulation_from_json
+from ..core.stub import TaskStub, TaskStubData
+from ... import log
+from ...components.data.sim_data import SimulationData
+from ...components.data.monitor_data import ModeSolverData
+from ..core.types import TaskType
+from ...components.simulation import Simulation
+from ...plugins.mode.mode_solver import ModeSolver
+
+
+class Tidy3dStub(BaseModel, TaskStub):
+
+    component: Union[Simulation, ModeSolver]
```

Comment:
After the 3.0 refactor, I'm guessing `Simulation` will be renamed to `FDTD` or something like that, right? Wouldn't it be more descriptive to have this field named `simulation` than `component`?

Reply:
I think we haven't decided this yet but you're probably right. On the one hand, we could wait until 3.0 to change the naming, but maybe `simulation` is best also because it is consistent with the webapi as it stands so I'm ok with changing it.

Reply:
There is one Enum named TaskType at GUI: 

class TaskType:
 FDTD, MODE_SOLVER, SWEEP, HEAT.
 
 we could wait until 3.0 to change the naming.

Reply:
yea let's not change this naming, but I think rather this is about what to call the `Tidy3dStub.component` field name. probably changing that to `Tidy3dStub.simulation` makes some sense, let's go with that.

Reply:
`Tidy3dStub.component` changed to `Tidy3dStub.simulation`.